config/config.json

```diff
@@ -465,6 +493,18 @@
           "dc_price_factor" : "3",
           "dh_price_factor" : "2",
           "trac_price_in_base_currency" : "0.4"
+        },
+        {
+          "blockchain_title": "OriginTrail-Parachain",
+          "network_id": "parachain:mainnet",
+          "blockchain_id": 2160,
```

Comment:
We will be using blockchain_id:2161 for parachain mainnet

Reply:
Let's update this when mainnet parachain is ready